### PR TITLE
Improving Stompy Walk Training Pipeline

### PR DESCRIPTION
**Pull request by michael-lutz**
- Fixed outstanding clarity issues from previous PR
- Added way to specify different policy & value network parameters
- Added (slightly buggy) LQR implementation for balancing
- Added way to render trajectories from cluster (assumes OpenGL access / Mesa)

Most notably, Stompy (with the stompy_walk.pth weights included) is now able to balance and maximize the height reward). This task is surprisingly difficult to train, and it can be greatly beneficial for bootstrapping an end-to-end goal-conditioned walking policy.

https://github.com/kscalelabs/ksim/assets/43460304/3ad489dd-6b83-4453-9461-f65f9f017c28

This was achieved with the following network sizes:
policy_hidden_layer_sizes: [64, 64, 64, 64, 64]
value_hidden_layer_sizes: [256, 256, 256, 256, 256]

Future runs might build upon these weights while lowering the control cost penalty and heavily increasing the forward reward weight.

### CODE REVIEW

**Comment by codekansas** [comment]
sweet :)

File: ksim/assets/stompy/lqr_balance_test.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
+import mujoco
```
Comment (budzianowski) [inline ksim/assets/stompy/lqr_balance_test.py]: let's keep experimental stuff in ksim/experimental/

Comment (budzianowski) [inline ksim/assets/stompy/lqr_balance_test.py]: Also, mpc - yay!!

File: ksim/mjx_gym/envs/stompy_env/rewards.py [hunk 2]
```diff
@@ -74,12 +75,12 @@ def forward_reward_fn(
     Returns:
         A float wrapped in a jax array.
     """
-    xpos = state.subtree_com[1][1]  # TODO: include stricter typing than mjxState to avoid this type error
+    xpos = state.subtree_com[1][1]  # Dimension 1 is the backward direction in the Stompy environment
```
Comment (budzianowski) [inline ksim/mjx_gym/envs/stompy_env/rewards.py:78]: Thanks for that!

File: ksim/mjx_gym/play.py [hunk 3]
```diff
@@ -58,11 +58,20 @@ def normalize(x: np.ndarray, y: np.ndarray) -> np.ndarray:
         normalize = (
             running_statistics.normalize
         )  # NOTE: very important to keep training & test normalization consistent
+    print(env.observation_size)
```
Comment (budzianowski) [inline ksim/mjx_gym/play.py]: Remove or add verbose option

**Review by budzianowski — COMMENTED** [review]
Great to see this progress! We will copy isaac reward setup (https://github.com/kscalelabs/sim/blob/ea1e46e635575ec36de3160487e8507ea105d573/sim/humanoid_gym/envs/stompy_config.py#L181) and we should be going (pun intended)

File: ksim/mjx_gym/play.py (hunk 3 above)
Comment (michael-lutz) [reply ksim/mjx_gym/play.py]: done!

**Review by budzianowski — APPROVED** [review]

**Pull request merged**